### ISSUE TITLE
Admin Menu Re-Shuffle 2

### ISSUE THREAD

**Issue by Tangol** [villagedefrance/OpenCart-Overclocked#130]
I've noticed that the **Extensions** link got moved after System.
It's place is after Tools as extensions are closely related to tools (see OpenBay which is a mod, located inside both tools and extensions).

![menu](https://user-images.githubusercontent.com/3336728/36355642-1a6a67d0-14ef-11e8-9428-ea2b1e16d2a1.jpg)

Ref https://github.com/villagedefrance/OpenCart-Overclocked/issues/117 and https://github.com/villagedefrance/OpenCart-Overclocked/commit/8e25d8e9e686a8e2f772f25a7f5bce5852dd0607

Thanks!

**Comment by villagedefrance** [comment]
Yes, I decided to change this for a simple reason: responsive admin pages.
(and that's also why I moved OpenBay Pro link out of Extensions)

The thing is, Extensions is the only top link in the menu that doesn't contain sub-levels with children. That means it will not overflow to the right on smaller screens, and although I 100% agree with your logic to have it next to "Tools", for practical reasons it should remain where it is for now.

The only way to put it back were it was is to create a responsive vertical menu. This is on my list of major things to do but not just yet.
